client-app/src/pages/account/views/AccountOrders.vue

```diff
@@ -111,5 +154,28 @@ export default class AccountOrders extends Vue {
     this.selectedOrderId = orderId;
     this.$bvModal.show("orderDetailsModal");
   }
+
+  dateChanged() {
+    if (this.startDate && this.endDate) {
```

Comment:
for any sort of validation, I would like to suggest use this plugin  https://bootstrap-vue.js.org/docs/reference/validation/
you can use template-based validation rules (end must be Greater that start date ) without any code in controller level.